pkgs/development/python-modules/remctl/default.nix

```diff
@@ -0,0 +1,12 @@
+{ lib, buildPythonPackage, isPy3k, pkgs, pytest, pytestrunner, typing }:
+
+buildPythonPackage {
+  inherit (pkgs.remctl) pname version meta;
+  src = pkgs.remctl.pythonsrc;
+  postPatch = ''
+    sed -i '/install_requires/d' setup.py
+  '';
+  buildInputs = [ pkgs.remctl ];
+  checkInputs = [ pytest pytestrunner ];
+  propagatedBuildInputs = if (!isPy3k) then [ typing ] else [ ];
```

Comment:
```suggestion
  propagatedBuildInputs = lib.optionals isPy2k)[ typing ];
```

Reply:
I don't think isPy2k exists?

```
$ rg -m1 isPy2 pkgs/top-level/python-packages.nix
21:  inherit (python.passthru) isPy27 isPy35 isPy36 isPy37 isPy38 isPy39 isPy3k isPyPy pythonAtLeast pythonOlder;
````

Reply:
I confused it with isPy27 which is practically the same.